Makefile

```diff
@@ -0,0 +1,35 @@
+# kernel-style V=1 build verbosity
+ifeq ("$(origin V)", "command line")
+	BUILD_VERBOSE = $(V)
+endif
+
+ifeq ($(BUILD_VERBOSE),1)
+	Q =
+else
+	Q = @
+endif
+
+export GO15VENDOREXPERIMENT:=1
```

Comment:
I agree, but for now the `.travis.yml` file includes go 1.5.4 in the language version matrix. Will probably drop support for that soon.